content/blog/2022-05-05-newsletter.es.md

```diff
@@ -0,0 +1,196 @@
+---
+title: The XMPP Newsletter April 2022
+author: XMPP Communication Team and Contributors
+date: 2022-05-05
+categories: ['Newsletter']
+---
+
+_¡Este articulo fue traducido por daimonduff y TheCoffeMaker, de la comunidad XMPP!_
+
+Bienvenidos al boletín de XMPP, ¡es un placer tenerles aquí de nuevo! Este número cubre el mes de abril de 2022.
+
+Al igual que este boletín, muchos proyectos y sus esfuerzos en la comunidad XMPP son el resultado del trabajo voluntario de la gente. Si estás contento con los servicios y el software que utilizas, especialmente en la situación actual, ¡considera dar las gracias o ayudar a estos proyectos! ¿Interesado en apoyar al equipo del Boletín? Lea más en la parte inferior.
+
+### Traducciones del Boletín de Noticias
+
+Este es un esfuerzo comunitario, y nos gustaría agradecer a los traductores sus contribuciones. ¡Los voluntarios son bienvenidos! Las traducciones del boletín XMPP se publicarán aquí (con cierto retraso):
+
+- Francés: [jabberfr.org](https://news.jabberfr.org/category/newsletter/) y [linuxfr.org](https://linuxfr.org/tags/xmpp/public)
```

Comment:
Maybe add the English original

Reply:
Oh! ... I did left Spanish instead of English, my bad!